routes/index.js

```diff
@@ -1,7 +1,23 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-
+const Movie = require("../models/Movie.model.js");
 /* GET home page */
-router.get('/', (req, res, next) => res.render('index'));
+router.get("/", (req, res) => res.render("index"));
+
+router.get("/movies", (req, res) => {
+  Movie.find({}).then((allMovies) => {
+    //console.log("Your movies are: ", allMovies);
+    res.render("movies", { allMovies });
+  });
+});
+
+router.get("/movies/:_id", (req, res) => {
+  const desiredMovie = req.params._id;
```

Comment:
Try being more descriptive of the variable names for what is stored inside. In this case you can name your variable **movieId** since you know it holds the id number. This might sound silly but in a big project it will help you a lot for code readability.